creating_citation_counts_tsv/scripts/create_citations_tsv.sh

```diff
@@ -90,5 +104,5 @@ cat creating_citation_counts_tsv/data/gene_info | awk '{if ($1 == 10116) print;}
 rm creating_citation_counts_tsv/data/gene_info
 
 # Lastly create the tsv with the total citations per gene along with the gene's information
-python3 creating_citation_counts_tsv/scripts/summarize_gene_citations_all_species.py
-
+python3 creating_citation_counts_tsv/scripts/summarize_gene_citations_all_species.py "creating_citation_counts_tsv/data/recent_pmid_year.ssv" recent $days_in_timeframe
+python3 creating_citation_counts_tsv/scripts/summarize_gene_citations_all_species.py "creating_citation_counts_tsv/data/past_pmid_year.ssv" past $days_in_timeframe
```

Comment:
refactor idea: 
instead of outputing 2 files, put all the data into the same TSV with new columns:
`days_in_timeframe`, `recent_timeframe_citation_count`, `past_timeframe_citation_count`

(working on this now)